feature/mypage/src/main/java/com/teamoffroad/feature/mypage/presentation/GainedCouponViewModel.kt

```diff
@@ -19,61 +18,76 @@ class GainedCouponViewModel @Inject constructor(
     private val userCouponRepository: UserCouponRepository,
 ) : ViewModel() {
 
-    private val _getUserAvailableCouponListState =
-        MutableStateFlow<UiState<UserAvailableCoupons>>(UiState.Loading)
-    val getUserCouponListState = _getUserAvailableCouponListState.asStateFlow()
+    private val _availableCouponListState =
+        MutableStateFlow(AvailableCouponListUiState())
+    val availableCouponListState = _availableCouponListState.asStateFlow()
 
-    private val _getUserUsedCouponListState =
-        MutableStateFlow<UiState<UserUsedCoupons>>(UiState.Loading)
-    val getUserUsedCouponListState = _getUserUsedCouponListState.asStateFlow()
+    private val _usedCouponListState =
+        MutableStateFlow(UsedCouponListUiState())
+    val usedCouponListState = _usedCouponListState.asStateFlow()
 
     private val _availableCouponsCount = MutableStateFlow<Int>(0)
     val availableCouponsCount = _availableCouponsCount.asStateFlow()
 
     private val _usedCouponsCount = MutableStateFlow<Int>(0)
     val usedCouponsCount = _usedCouponsCount.asStateFlow()
 
-    private val _userAvailableCoupons = MutableStateFlow<List<UserAvailableCoupons.AvailableCoupons>>(emptyList())
+    private val _userAvailableCoupons =
+        MutableStateFlow<List<UserAvailableCoupons.AvailableCoupons>>(emptyList())
     val userAvailableCoupons = _userAvailableCoupons.asStateFlow()
 
     private val _userUsedCoupons = MutableStateFlow<List<UserUsedCoupons.UsedCoupons>>(emptyList())
     val userUsedCoupons = _userUsedCoupons.asStateFlow()
 
     fun getUserAvailableCoupons(isUsed: Boolean, cursorId: Int) {
+        _availableCouponListState.value = _availableCouponListState.value.copy(
```

Comment:
```suggestion
        _availableCouponListState.value = availableCouponListState.value.copy(
```
언더바가 붙은 친구와 붙지 않은 친구의 차이점을 아시나요?

Reply:
언더바 붙은 친구는 private으로 설정되어 뷰모델 내에서만 접근 가능해 외부에서는 데이터 수정을 못합니다. 언더바 붙지 않은 친구는 외부에서 접근해 데이터를 읽을 수 있습니다.